src/test/scala/chiselTests/Assert.scala

```diff
@@ -61,6 +61,17 @@ class BadUnescapedPercentAssertTester extends BasicTester {
   stop()
 }
 
```

Comment:
It would be helpful to add some tests for `assume` as well.

Reply:
@jackkoenig  cool - let me create a new Assume.scala file and add to that one! 

Also - do I need to do anything different to add these tests in CI or will it automatically get picked up?

Reply:
they will get picked up as long as they are in the same file and extend the scala test as the others do

Reply:
@jackkoenig  @mwachs5 created a spec for Assume similar to that for Assert (in fact first version is just a copy and replace assert with assume - since as per spec in simulation they should be identical). 

But looks like for some reason `assume(false.B)` does not trigger a failure while `assert(false.B)` does. 

This warning could potentially be the reason 
```
------------------------------------------------------------------------------
Warning: 1 verification statements (assert, assume or cover) were removed 

when compiling to Verilog because the 

basic Verilog standard does not support them.

 If this was not intended, compile to System Verilog instead using the `-X sverilog` compiler flag.
------------------------------------------------------------------------------
```

Reply:
Well i guess for the assume test we need to emit to SV instead of verilog?

Reply:
Yes - confirmed that it is indeed the reason by inspecting the verilog. Let me try changing the compiler first if that is straightforward - else will change the test to just inspect the generated FIRRTL/verilog

@jackkoenig - feel free to share specific pointers to change the compiler (like by passing an annotation or something).

Reply:
@mwachs5  - not sure if your suggestion is same as that of @jackkoenig - I am assuming  it is! (Pun intended)

Reply:
@girishpai it specifies the compiler here: https://github.com/chipsalliance/chisel3/blob/2b48fd15a7711dcd44334fbbc538667a102a581a/src/main/scala/chisel3/testers/TesterDriver.scala#L57

Note that changing this will have pretty wide ranging impacts so might affect lots of other tests. It's probably better to do something that just checks the emitted FIRRTL like so: https://github.com/chipsalliance/chisel3/blob/6d4e6edc9a62c166af95b41b6aea9634d34275ed/src/test/scala/chiselTests/BulkConnectSpec.scala#L20

Reply:
@jackkoenig  thanks! Most likely will go with inspecting FIRRTL - but for future purposes might be a good idea to provide support to override the Emitter at the Spec/test level.